### ISSUE TITLE
In some apparently random circumstances, a second input cursor appears and writing is duplicated

### ISSUE THREAD

**Issue by max-l** [Zettlr/Zettlr#471]
Every once in a while, for no apparent reason, there is another cursor that appears and whatever I write gets written twice, once for each cursor. I might be accidentally hitting special keys to turn this on, but I'm not aware of it. I am not even able to reproduce it. 

When it happens, I switch to another file, and come back, and the second cursor disappears.

probably a duplicate of this issue: https://github.com/Zettlr/Zettlr/issues/222

+ OS and version: Ubuntu 18.04.3 LTS
+ Zettlr Version: 1.4.3 
+ Screen Resolution: 1920 x 1080

**Comment by nathanlesage** [comment]
Hey, thanks for reporting. So, you cannot remove the second cursor by clicking somewhere else in the text? It just stays?

**Comment by Zverik** [comment]
macOS, Zettlr 1.5.0: by clicking with Cmd anywhere on text, you can add more and more cursors with behaviour described here. Clicking anywhere else resets cursors to just one.

**Comment by nathanlesage** [comment]
@Zverik That is exactly how it is supposed to be. One Windows, the same can be achieved by `Ctrl`-clicking.

**Comment by Zverik** [comment]
Thanks Nathan, great to know this is not a bug.

But at the same time, is it that frequently needed operation to bind such a common key to it? Currently on macOS I Ctrl-click to open a link, and Cmd-click for... adding text in two places simultaneously? To me, it would be much more obvious if these key modifiers were switched: Ctrl on macOS is supposed to be seldom-pressed button.

**Comment by nathanlesage** [comment]
Well, yes, I had some confusion going on concerning the keybindings, because I added all possible key bindings regardless of platform, which was pretty much a mess. But I cleaned it up for 1.6 so that all these `Ctrl`-keys only bind on Windows and Linux, and no longer on macOS. Have a look at the current beta for testing it out!

That being said, the shortcut for opening links is `Option` or `Alt`, depending on Platform. That `Ctrl` also works is because on some Linux platforms, apparently `Alt`-click initiates dragging the window around, so these can fall back to `Ctrl`. But the major shortcut for opening links is `Alt`.

I will not override the default `Cmd/Ctrl`-behaviour in this case (which is built-into CodeMirror), because it is actually quite easy to not trigger that behaviour. I have to admit that I have a hard time understanding how some people accidentally trigger that behaviour in the first place :/
